pkg/services/auth/auth_token.go

```diff
@@ -35,6 +35,19 @@ func (s *UserAuthTokenService) Init() error {
 	return nil
 }
 
+func (s *UserAuthTokenService) ActiveAuthTokenCount() (int64, error) {
+
+	tokenMaxLifetime := time.Duration(s.Cfg.LoginMaxLifetimeDays) * 24 * time.Hour
```

Comment:
Duplicate code which are used in `LookupToken` and almost also in token_cleanup.go. Should be able to extract some common parts.